### DISCUSSION TITLE
buddhism/2024/quotes/

### DISCUSSION THREAD

**Discussion by giscus** [shane0/buddhism#80] [General]
# buddhism/2024/quotes/

http://shanenull.com/buddhism/2024/quotes/

**Comment by shane0** [comment]
Trekcho. https://shane0.pythonanywhere.com/meditation/meditation/42

**Reply by shane0** [reply]
https://shanenull.com/buddhism/2021/docs_2021/sadhana/sunset/

**Comment by shane0** [comment]
Probing to the root of mind means investigating which of the three doors (of body, speech and mind) it is that causes us to wander throughout beginningless time in saṃsāra and which it is that carries out virtuous or non-virtuous actions. When investigating, we discover mind to be the most important factor. Searching for hidden flaws means examining whether body, speech and mind are unitary or distinct, and finding that, while on a conventional level they appear to be related, ultimately there is no real entity called ‘mind’ that could be one with or distinct from anything else. It is simply a deception, a clear appearance of something unreal. When you investigate the essence of this mind, even if you search for its arising you cannot find it. There is no reality to mind’s apparent presence. Nor is there anywhere that it ceases. It is thus without foundation or origin. When investigating whether the searching mind and the mind that is sought for are the same or different, it seems as if one gives rise to the other. But as the mind that is the object of the search is unreal, so too is the mind that searches. Nevertheless, by clinging to a self in all our vague and transitory thoughts, which are brought about by fleeting causes and conditions, we experience the delusion of saṃsāric existence.

**Comment by shane0** [comment]
Anything changing is not real

**Reply by shane0** [reply]
signless
http://shanenull.com/buddhism/2023/sanskrit/#signlessness
